lti_consumer/plugin/views.py

```diff
@@ -349,7 +349,7 @@ def perform_create(self, serializer):
         renderer_classes=[LineItemResultsRenderer],
         content_negotiation_class=IgnoreContentNegotiation,
     )
-    def results(self, request, user_id=None, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Why this was removed?

Reply:
pylint is complaining about a useless suppression here. Since I upgraded dependencies, the lint rules might have changed.